addons/mail/static/src/models/rtc_session/rtc_session.js

```diff
@@ -241,12 +238,17 @@ registerModel({
          */
         _debounce(f, delay) {
             this._timeoutId && browser.clearTimeout(this._timeoutId);
-            this._timeoutId = browser.setTimeout(() => {
-                if (!this.exists()) {
-                    return;
-                }
-                f();
-            }, delay);
+            this.update({
+                _timeoutId: browser.setTimeout(
+                    () => {
+                        if (!this.exists()) {
+                            return;
+                        }
+                        f();
+                    },
```

Comment:
I would still make a method for those, it's possible to pass the function as param to the method with `.bind(this, f)` when registering the timeout